index.html

```diff
@@ -0,0 +1,207 @@
+<!doctype html>
+<html>
+<head>
+<title>Nicholas Botorog</title>
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link rel="stylesheet" href="style.css" >
```

Comment:
Optional addition here would be to add a personal favicon via an additional link tad.  Also, link tags are really powerful, and can also be used to preload resources for better performance.  Might be an idea to read about this in the mdn docks when you have some time.